SPEC.md

```diff
@@ -121,7 +121,15 @@ The test index makes use of symbolic language in describing connection and messa
 
 ### ZG-CONFORMANCE-006
 
-    The node requests mtGET_PEER_SHARD_INFO_V2 after start up.
+    The node requests mtGET_PEER_SHARD_INFO_V2 after connection and handshake.
 
+    <>
+    <-> handshake
```

Comment:
```suggestion
```
`<>` implies a complete handshake

Reply:
I thought this only implies connection. How to represent 'no handshake'?

Reply:
Done